src/foam/classloader/WebModelFileDAO.js

```diff
@@ -52,10 +52,19 @@ foam.CLASS({
         var model;
         var foamCLASS = foam.CLASS;
 
-        foam.CLASS = function(m) {
-          var cls = m.class ? foam.lookup(m.class) : foam.core.Model;
-          model = cls.create(m, self);
-          foam.CLASS = foamCLASS;
+        foam.CLASS = function(clsDef) {
+          var classOfModel = clsDef.class ? foam.lookup(clsDef.class) :
```

Comment:
It's wordier, but it describes the meaning of the model/class back-flips we're doing here.

@jacksonic @mcarcaso @shepheb @kgrgreer am I the only one that finds this easier to parse? If so, I'll happily change it back, and maybe add a short comment instead.